src/deno_dir.rs

```diff
@@ -165,12 +165,27 @@ impl DenoDir {
 
     let j: Url =
       if containing_file == "." || Path::new(module_specifier).is_absolute() {
-        Url::from_file_path(module_specifier).unwrap()
+        let r = Url::from_file_path(module_specifier);
+        // TODO(ry) Properly handle error.
+        if r.is_err() {
+          error!("Url::from_file_path error {}", module_specifier);
```

Comment:
I'm not sure what's the practical difference with the situation prior to this patch.
But it's not worse either. 🤷‍♂️

Reply:
It sometimes would fail on windows due to not having "C:/" ... and it wouldn't print anything.